### DISCUSSION TITLE
Importer only importing partial recipes from NextCloud Cookbook

### DISCUSSION THREAD

**Discussion by quamok** [TandoorRecipes/recipes#559] [Q&A]
Hello there, this looks like a really nice project. However I can't seem to import all my recipes from NextCloud Cookbook. 

I have approximately 150 recipes, but by trying to import them into Tandoor Recipes it only imports around 50. They all seem to have the same structure, as described in the Tandoor documentation. If I try to run the importer again on the zip file, I get "0 recipes imported". 

Is there a way to troubleshoot what could be wrong with the 2/3 of the recipes that aren't getting imported? Is there anyone else having issues or success stories importing from NextCloud Cookbook?

Thanks!

**Comment by vabene1111** [comment]
hey, this sounds like a bug so i will move it to the issues. Some questions

- The importer checks for certain file paths to prevent recognizing images or similar as recipes. The recipes that arre not imported, do they contain special characters, other file endings or similar ? 
- Does the import log also only show the 50 that are imported or does it show all of them ?

can you provide at least one of the files (including its path inside the zip) that does not work ? 

Thanks for providing the data

**Reply by vabene1111** [reply]
EDIT: apparently i cant move this to an issue so lets stay here for now

**Comment by quamok** [comment]
They don't seem to contain special characters, though some are in french, so would contain accents. But a lot of non-imported recipes are in english, so I guess that couldn't be the issue.

The log only shows the imported recipes.
Example: 5 recipes imported. The log shows:

_1 - baklavas 
Imported 1 recipes._

Here's a link to a test file containing 5 recipes. Only baklavas is imported out of the 5. https://send.bitwarden.com/#FMlloMO03U-PLq0MAMy9RQ/wFFO9l09sCRLsMZHMRgKQw

Thank you for your help!

**Reply by vabene1111** [reply]
awesome, i will create an issue and see what can be done about this

**Reply by vabene1111** [reply]
@quamok i am sorry, i just got to fixing this and noticed the download link has expired. could you send me a new one ? 

if you dont want to share this publicly you can send it to info@tandoor.dev
